slides/_code_structure.qmd

```diff
@@ -0,0 +1,106 @@
+# Structuring your code
+
+<!--=================================================================================-->
+
+## Functions {.smaller}
+
```

Comment:
Mention [pasta antipatterns](https://www.techtarget.com/searchsoftwarequality/tip/Fix-spaghetti-code-and-other-pasta-theory-antipatterns) in code?
spaghetti, lasagna, and ravioli at least.
The "pizza code" is new to me.